olm_deploy/operatorregistry/registry-deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           value: ${IMAGE_ELASTICSEARCH6}
         - name: IMAGE_LOGGING_KIBANA6
           value: ${IMAGE_LOGGING_KIBANA6}
+        - name: IMAGE_LOGGING_LOGFILEMETRICEXPORTER
```

Comment:
Please modify to https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml#L120  `IMAGE_LOG_FILE_METRIC_EXPORTER`